ref_kernels/ind/bli_gemm1m_ref.c

```diff
@@ -34,10 +34,10 @@
 
 #include "blis.h"
 
-#undef  GENTFUNCCO
-#define GENTFUNCCO( ctype, ctype_r, ch, chr, opname, arch, suf ) \
+#undef  GENTFUNCRO
+#define GENTFUNCRO( ctype_r, ctype, chr, ch, opname, arch, suf ) \
```

Comment:
@devinamatthews I'm trying to make sense of why this changed from `CO` to `RO`, and why the function name now user `chr`. 🤔  Halp.